test_sets/webengine_projection_mode.txt

```diff
@@ -6,3 +6,12 @@
 ./test_scripts/WebEngine/WebEngineProjection/006_Registration_reject_WEB_VIEW_when_PT_is_empty.lua
 ./test_scripts/WebEngine/WebEngineProjection/007_Registration_reject_WEB_VIEW_is_absent_in_PT.lua
 ./test_scripts/WebEngine/WebEngineProjection/008_Registration_appHMIType_is_array.lua
+./test_scripts/WebEngine/WebEngineProjection/008_Registration_reject_WEB_VIEW_app_is_absent_in_PT_WEB_VIEW_part_of_collection.lua
```

Comment:
@YarikMamykin Please re-enumerate test scripts

Reply:
@aderiabin fixed in https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/32/commits/66e780b7fb2d744f4ac6fd6ee635b18b1bf5b68f